go.mod

```diff
@@ -34,7 +38,7 @@ require (
 	k8s.io/component-base v0.27.4
 	k8s.io/klog/v2 v2.90.1
 	k8s.io/utils v0.0.0-20230209194617-a36077c30491
-	sigs.k8s.io/cluster-api v0.0.0-00010101000000-000000000000
+	sigs.k8s.io/cluster-api v1.5.1-0.20230825144253-1c45db7b5e28
```

Comment:
Will roll this back once we get a release of CAPI v1.5.1. Hold is on until then.